lib/http_server_fixture/handler/handler.ex

```diff
@@ -0,0 +1,4 @@
+defmodule HTTPServerFixture.Handler do
+  alias HTTPServer.Request
+  @callback handle(req :: %Request{}) :: {status_code :: integer(), body :: term()}
+end
```

Comment:
this is actual http_server code. I suggest placing it in the `HTTPServer` module, and not the fixture.

Looks great though!

Reply:
`handle` is now in the `HTTPServer.Handler` module!

Reply:
🎉